src/main/java/ca/nrc/cadc/beacon/web/view/StorageItem.java

```diff
@@ -192,6 +200,24 @@ public String getOwner()
         return owner;
     }
 
+    public String getOwnerCN()
```

Comment:
Please create a good unit test for this method.  I see plenty of opportunity for NullPointerExceptions, plus it should check the order of CN with '_', CN as-is, and DN as fallback.